src/main/java/org/usfirst/frc/team2473/framework/CHS_SparkMax.java

```diff
@@ -0,0 +1,46 @@
+package org.usfirst.frc.team2473.framework;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class CHS_SparkMax {
+	private CANSparkMax spark;
+	private CANEncoder encoder;
+
+	private double lastEncoderValue = 0;
+
+	public CHS_SparkMax(int port, MotorType motorType) {
+		spark = new CANSparkMax(port, motorType);
+		encoder = spark.getEncoder();
+	}
+
+	public CANSparkMax getSparkMaxObject() {
+		return spark;
+	}
+
+	public void set(double power) {
+		spark.set(power);
+	}
+
+	public void stop() {
+		spark.set(0);
+	}
+
+	public double getEncoderPosition() {
```

Comment:
This is fairly awkward, the encoder is being relegated to a second class citizen, i.e.:

Motor:
set()
stop()

Encoder:
getEncoderPosition()
getEncoderVelocity()